src/models/report.js

```diff
@@ -0,0 +1,16 @@
+import Config from '../config';
+import entries from 'object.entries';
+import { apiUrl } from '../shared/util';
+
+class Report {
+  constructor(props) {
+    entries(props).forEach(([k, v]) => this[k] = v);
+  }
+}
+
+Report.getKDReport = (ajaxGet, token: string, id: number)  => {
+  const url = apiUrl(`${Config.magasin.urls.storagefacility.baseUrl(id)}/report`);
+  return ajaxGet(url,token).map(response => response && new Report(response));
```

Comment:
som følge av merge med master må du nå dekonstruere argumentet til map for å få ut respons aka:
``` .map(({ response }) => response && new Report(response)); ```